USERGUIDE.md

```diff
@@ -244,9 +255,9 @@ For more information on how to use Grafana, see the [official Grafana user guide
 
 [Apache Superset](https://superset.incubator.apache.org) provides some interesting features to explore your data and build basic dashboards.
 
-Head to the Superset interface, if you are using **minikube**, you can use the following command :
+Head to the Superset web interface by taping in your browser the `superset.traefikIngress.host`. E.g. :
 ```
-minikube service -n fadi fadi-superset
+http(s)://superset.test.local
```

Comment:
* should be a clickable url, not sure why the (s), I'd suggest to remove it if both work
* taping -> typing